src/Appliance_Item.php

```diff
@@ -58,7 +58,7 @@ public static function getTypeName($nb = 0)
     }
 
 
-    public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
+    public function getTabNameForItem(CommonGLPI $item, $withtemplate = false)
```

Comment:
I am not sure of that, but I think that `$withtemplate` could be `2` here. See https://github.com/glpi-project/glpi/blob/1f1ae81e68c01b2d715264161f88500118e05dcf/src/CommonDBTM.php#L483

Reply:
Indeed.. Then, fixes already done on plugin CS are wrong as well :/